source/parameters/patch.md

```diff
@@ -2,5 +2,123 @@
 (s:patch.params)=
 ## PatchP:  Patch Parameters
 
-In Construction...
+```{figure} figures/patch.svg
+:name: f:patch
 
+A) A `Patch` element can align its exit face arbitrarily with respect to its entrance face. The
+red arrow illustrates a possible particle trajectory form entrance face to exit face. B) The
+reference length of a `Patch` element, if `ref_coords` is set to the default value of
+`exit_end`, is the longitudinal distance from the entrance origin to the exit origin using the
+reference coordinates at the exit end as shown. If `ref_coords` is set to `entrance_end`, the
+length of the patch will be equal to the `z_offset`.
+```
+
+The `PatchP` parameter group describes the shift the reference curve of a `Patch` or element.
+Also see [`ReferenceChangeP`](#s:ref.change.params) for the parameter group that can be used
+to change reference time and/or energy. 
+The components of `PatchP` are:
+```{code} yaml
+  x_offset          - Offset in x-direction.
+  y_offset          - Offset in y-direction.
+  z_offset          - Offset in z-direction.
+  x_rot             - Offset in x-direction.
+  y_rot             - Offset in x-direction.
+  z_rot             - Offset in x-direction.
```

Comment:
The descriptions on the right were copy-pasted and need to be updated.

Reply:
Done.